apps/Spruce Connect.src

```diff
@@ -296,13 +305,19 @@ def child_schedules(dni){
 	def childDevice = childDevices.find{it.deviceNetworkId == dni}
     log.debug "${childDevice}"
     
-    if(settings.pause == true) childDevice.createScheduleDevices("${app.id}", 99, 0, "Spruce Pause")
+    // Pause Device for this Controller
+	if(settings.pause == true) {
+		String controllerName = settings.controller.displayName
```

Comment:
I see error displayName does not exist.  settings.controller should be all that is needed.  Error is causing schedule and pause child devices not to populate